package.json

```diff
@@ -41,6 +41,7 @@
     "light-server": "^1.0.3",
     "mocha": "^2.4.5",
     "nyc": "^6.4.4",
+    "prop-types": "^15.5.9",
```

Comment:
I think that since it is used in the source code directly, it should be in `dependencies`, not `devDependencies`. Some tools complain about it:
 
![image](https://user-images.githubusercontent.com/577441/46143919-d370d600-c25b-11e8-97f9-ac5404e40b33.png)